
(node:891160) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_2',
  containerIp: '10.0.3.194',
  mitmPort: 1001,
  autoAccessFixed: 1
}
2022-10-25 19:25:11.880 - [Info] MITM Version: 2
2022-10-25 19:25:11.881 - [Info] Auto Access Enabled: true
2022-10-25 19:25:11.881 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 19:25:11.888 - [Info] SSH man-in-the-middle server for 10.0.3.194 listening on 10.0.3.1:1001
(node:903017) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_2',
  containerIp: '10.0.3.202',
  mitmPort: 1001,
  autoAccessFixed: 1
}
2022-10-25 19:26:23.437 - [Info] MITM Version: 2
2022-10-25 19:26:23.438 - [Info] Auto Access Enabled: true
2022-10-25 19:26:23.438 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 19:26:23.445 - [Debug] listen EADDRINUSE: address already in use 10.0.3.1:1001
2022-10-25 19:26:23.445 - [Error] Another MITM instance or another program is already listening on this port
{"level":"error","message":"Forever detected script exited with code: 1"}
2022-10-25 19:26:23.871 - [Debug] [Connection] Attacker connected: 222.134.240.91 | Client Identification: SSH-2.0-Go
2022-10-25 19:26:24.030 - [Debug] [Connection] Attacker connected: 112.160.137.62 | Client Identification: SSH-2.0-Go
2022-10-25 19:26:24.277 - [Debug] [Connection] Attacker connected: 78.187.13.206 | Client Identification: SSH-2.0-Go
2022-10-25 19:26:24.714 - [Debug] [Connection] Attacker connected: 5.28.139.161 | Client Identification: SSH-2.0-Go
2022-10-25 19:26:25.199 - [Debug] [Auth] Attacker 78.187.13.206 trying to authenticate with 'none'
2022-10-25 19:26:25.200 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:26:25.304 - [Debug] [Auth] Attacker 112.160.137.62 trying to authenticate with 'none'
2022-10-25 19:26:25.304 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:26:25.406 - [Debug] [Connection] Attacker connected: 78.189.25.224 | Client Identification: SSH-2.0-Go
2022-10-25 19:26:25.492 - [Debug] [Auth] Attacker 78.187.13.206 trying to authenticate with 'password'
2022-10-25 19:26:25.492 - [Debug] [Auto Access] Attacker: 78.187.13.206, Threshold: 1, Attempts: 1
2022-10-25 19:26:25.493 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 19:26:25.493 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 19:26:25.551 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 19:26:25.552 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 19:26:25.552 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.194
2022-10-25 19:26:25.556 - [Debug] [Auth] Attacker 222.134.240.91 trying to authenticate with 'none'
2022-10-25 19:26:25.556 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:26:25.664 - [Debug] [Auth] Attacker 5.28.139.161 trying to authenticate with 'none'
2022-10-25 19:26:25.664 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:26:25.740 - [Debug] [Auth] Attacker 112.160.137.62 trying to authenticate with 'publickey'
2022-10-25 19:26:25.741 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 19:26:25.741 - [Debug] [Auth] Attacker: 112.160.137.62 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 19:26:26.016 - [Debug] [Auth] Attacker 5.28.139.161 trying to authenticate with 'publickey'
2022-10-25 19:26:26.016 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 19:26:26.017 - [Debug] [Auth] Attacker: 5.28.139.161 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 19:26:26.084 - [Debug] [Auth] Attacker 222.134.240.91 trying to authenticate with 'password'
2022-10-25 19:26:26.084 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 19:26:26.084 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.194
2022-10-25 19:26:26.180 - [Debug] [Connection] Attacker closed the connection
